content/reference/endpoints/rates_tables.md

```diff
@@ -1,7 +1,8 @@
 # Rates Tables
 
-1. TOC
-{:toc}
+## Overview
+
+This resource represents Rates Tables - a resource for grouping price-related resources (such as Rates Rules, Seasons, Periods) for [Rentals](/reference/endpoints/rentals/). Rates Tables can be shared between Rentals to make them reusable.
```

Comment:
```suggestion
This resource represents Rates Tables - a resource for grouping price-related resources (such as Rates Rules, Seasons, Periods) for [Rentals](/reference/endpoints/rentals/). Rates Tables can be shared between Rentals to make pricing setup between rentals reusable.
```

Also, there is a bit tricky logic on Core that handled changes to rates table on rental depending on the rates management of that rental (you can check Rental spec from line 548)
But basically if rental has external rates management and a `private` rates table, user will not be able to change this rates table to a different one or update it to a shared table

Reply:
@adamgrad can you suggest a change for it then that would cover all the cases?